managed/services/config/pmm-managed.yaml

```diff
@@ -19,7 +19,10 @@ services:
       GRAFANADB_SELECT:
         enabled: true
         timeout: 5s
-        db_file: /srv/grafana/grafana.db
+        use_separate_credentials: true
+        separate_credentials:
+          username: grafana
+          password: grafana
```

Comment:
This file has TODO comment on the top. Can we eliminate its extension and use just plain constants?

Reply:
As per TODO, all of them need to be migrated. For the sake of scope and testing, isn't it better to do that in one PR?

Reply:
Yes, it's ok to do that later. Just wanted to get your attention to plans regarding this config.